app/components/sessions/forms/edit-session.js

```diff
@@ -41,12 +41,12 @@ export default Component.extend({
   // eslint-disable-next-line ember/no-actions-hash
   actions: {
 
-    meetingNumberChangedAction(meetingNumber) {
+    meetingNumberChangedAction(event) {
+      const meetingNumber = event.target.value;
       const meetingYear = moment(this.get('meeting.plannedStart')).year();
       this.set('meetingNumber', meetingNumber);
       this.set('formattedMeetingIdentifier', `VR PV ${meetingYear}/${meetingNumber}`);
       this.set('numberRepresentation', this.get('formattedMeetingIdentifier'));
-      this.set('meeting.numberRepresentation', this.get('formattedMeetingIdentifier'));
```

Comment:
Deze set was de oorzaak van de hieruit gekomen bug.
Bij uiteindelijk saven doen we nog eens een set van die waarde, dus hier overbodig (en dirty zonder rollback)

Reply:
Aha, goed om te weten. Bedankt!